NotificationTrigger/index.js

```diff
@@ -0,0 +1,53 @@
+module.exports = async function(context, myTimer) {
+	const moment = require('moment-timezone')
+	const timeStamp = new Date().toISOString()
+
+	const { post } = require('./http')

```

Comment:
@shrijan-siris the names `post` and `http` are very generic, can you use more specific names for variables? probably something like `sendNotification`

Reply:
i think the function is more generic with post as we can reuse it will any other post request that might be needed